python/dask_cudf/dask_cudf/io/csv.py

```diff
@@ -62,16 +62,32 @@ def read_csv(path, chunksize="256 MiB", **kwargs):
     1  2  hello
     2  3     ai
     """
+
+    # Handle `chunksize` deprecation
+    if "chunksize" in kwargs:
+        chunksize = kwargs.pop("chunksize", "default")
+        warn(
+            "`chunksize` is deprecated and will be removed in the future. "
+            "Please use `blocksize` instead.",
+            FutureWarning,
+        )
+        if blocksize == "default":
+            blocksize = chunksize
+
+    # Set default `blocksize`
+    if blocksize == "default":
+        blocksize = "256 MiB"
+
     if "://" in str(path):
         func = make_reader(cudf.read_csv, "read_csv", "CSV")
-        return func(path, blocksize=chunksize, **kwargs)
+        return func(path, blocksize=blocksize, **kwargs)
     else:
-        return _internal_read_csv(path=path, chunksize=chunksize, **kwargs)
+        return _internal_read_csv(path=path, blocksize=blocksize, **kwargs)
 
 
-def _internal_read_csv(path, chunksize="256 MiB", **kwargs):
-    if isinstance(chunksize, str):
-        chunksize = parse_bytes(chunksize)
+def _internal_read_csv(path, blocksize="256 MiB", **kwargs):
+    if isinstance(blocksize, str):
+        blocksize = parse_bytes(blocksize)
```

Comment:
`parse_bytes` already handles the case that its argument is an `int`

Reply:
oops - good point